demo/index.html

```diff
@@ -0,0 +1,161 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <meta charset="utf-8">
+  <title>Cliquetis demo</title>
+  <style>
+  .log {
+    position: fixed;
+    top: 0;
+    right: 0;
+    width: 50%;
+    height: 100%;
+    padding: .4em;
+    font-size: .8em;
+    color: white;
+  }
+  </style>
+</head>
+<body>
+  <h1>Todo list</h1>
+  <p>Last modified: <span id="lastModified"></span></p>
+  <ul id="todo"></ul>
+  <form>
+    <p>
+      <input type="text" name="label" placeholder="Thing">
+      <input type="submit" value="Add">
+    </p>
+  </form>
+  <p>
+    <input type="button" id="clearCompletedBtn" value="Clear completed">
+    <input type="reset" id="clearAllBtn" value="Clear collection">
+    <input type="button" id="syncBtn" value="Synchronize collection">
+  </p>
+  <p>
+    <textarea id="log" class="log"></textarea>
+  </p>
+  <script src="../dist/cliquetis.js"></script>
+  <script>
+  function main() {
+    const db = new Cliquetis({remote: "http://0.0.0.0:8888/v0"});
```

Comment:
One cliquetis client should be tied to one major version of cliquet at one particular point in time. As such, it's not possible to have one version of cliquet specifying a different version than what it's able to handle, otherwise unexpected behaviours will happen.

For instance, with the upcoming changes in Cliquet, changing the version in cliquetis will not arrange anything, the client code needs to be updated, and then it will not be compatible with the old version of the APIs.